microservice-transaction-sample/README.md

```diff
@@ -27,20 +29,26 @@ The database is service-independent and exists for managing transaction metadata
 
 In this sample application, for ease of setup and explanation, we co-locate the coordinator database in the same Cassandra instance of the Order Service, but of course, the coordinator database can be managed as a separate database.
 
-### Schemas
+Also, note that application-specific error handling, authentication processing, etc., are omitted in the sample application since it focuses on explaining how to use Scalar DB.
+Please see [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions) for the details of how to handle exceptions in Scalar DB.
+
+Additionally, you assume each service has one container in this sample application to omit considering request routing between the services.
```

Comment:
```suggestion
Additionally, you assume each service has one container in this sample application to avoid considering request routing between the services.
```